solidity/dashboard/src/pages/liquidity/LiquidityPage.jsx

```diff
@@ -117,36 +117,22 @@ const LiquidityPage = ({ headerTitle }) => {
     }
   }, [isConnected, keepTokenBalance.value, hideBanner, isBannerVisible])
 
-  const addLpTokens = (
-    wrappedTokenBalance,
-    liquidityPairContractName,
-    pool,
-    awaitingPromise
-  ) => {
+  const addLpTokens = (poolId, wrappedTokenBalance, awaitingPromise) => {
+    const liquidityPairData = LIQUIDITY_REWARD_PAIRS[poolId]
```

Comment:
Although I really like the name of this const I think we should be consistent in the app. I think we usualy named it as `liquidityRewardPair`.

So either change this to `liquidityPairData` -> `liquidityRewardPair`, or change all occurrences of `liquidityRewardPair` -> `liquidityPairData`.

I like `liquidityPariData` name more, but the first case is faster to fix. The `liquidityPairData` is also reasonable variable name since it is singular from of `LIQUIDITY_REWARD_PAIRS`. I'll left it to you to decide.

Reply:
e34c63a